pmm/aws-staging-start.groovy

```diff
@@ -407,20 +407,25 @@ pipeline {
                             export PATH=\$PATH:/usr/sbin
                             test -f /usr/lib64/libsasl2.so.2 || sudo ln -s /usr/lib64/libsasl2.so.3.0.0 /usr/lib64/libsasl2.so.2
                             export CLIENT_IP=\$(curl ifconfig.me);
+                            sudo yum -y install https://repo.percona.com/yum/percona-release-latest.noarch.rpm
                             if [[ \$CLIENT_VERSION = dev-latest ]]; then
-                                    sudo yum -y install https://repo.percona.com/yum/percona-release-latest.noarch.rpm
-                                    sudo percona-release disable all
-                                    sudo percona-release enable original testing
-                                    sudo yum clean all
-                                    sudo yum -y install pmm2-client
-                                    sudo yum -y update
+                                sudo percona-release disable all
+                                sudo percona-release enable original testing
```

Comment:
Is it really needed to enable testing repo?

Reply:
yes because we want to test upgrade of client